### ISSUE TITLE
Test suite hangs

### ISSUE THREAD

**Issue by nwinch** [Prismatik/auth#28]
Tests seem to hang until you `ctrl-c` outta there.

**Comment by davidbanham** [comment]
Yeah what is _up_ with that? It seems like the rethink connection is staying open despite us explicitly draining it. I can't figure it out.
